microsoft/electionguard-ui
# 🗳️ Ballot Upload

### Issue
Fixes #129 

### Description
Updates the election list page to show new fields including number of ballots cast and also a button that can submit encrypted ballots.

### Testing
1. Click Manage Elections

Expected: new fields including 
- Name - which is extracted from the manifest
- Ballots - shows ballots submitted and you can mouse over to see number of spoiled and cast
- Action (blank header): with an upload button

Should look something like this:

![image](https://user-images.githubusercontent.com/1769905/156800216-c6fcd751-4dfc-43a2-9c5e-a510c07697aa.png)

2. Click the upload button and select an invalid encrypted file

Expected an error message should show like this:

![image](https://user-images.githubusercontent.com/1769905/156800634-4d954b98-3819-481e-92fb-4a16f71faef5.png)

3. Select a valid encrypted file

Expected: it should increase the number of ballots
